idtoken/validate.go

```diff
@@ -122,6 +122,36 @@ func Validate(ctx context.Context, idToken string, audience string) (*Payload, e
 	return defaultValidator.validate(ctx, idToken, audience)
 }
 
+// GetPayload parses the given token and returns its payload.
+//
+// Warning: This function does not validate the token prior to parsing it.
+//
+// GetPayload is primarily meant to be used to inspect a token's payload. This is
+// useful when validation fails and the payload needs to be inspected.
+//
+// Note: A successful Validate() invocation with the same token will return an
+// identical payload.
+func (v *Validator) GetPayload(ctx context.Context, idToken string) (*Payload, error) {
```

Comment:
nit: because no verification is needed I wonder if this should be a method at all. Maybe just a function is enough. The main reason for a method is incase users need to configure the underlying client for nextwork requests, but this will not make any, right?